### PR TITLE
Fix: zendesk logout

### PR DESCRIPTION
**Pull request by kiwis08**
## Summary
Fix uncaught exception when `ZendeskMessaging.logoutUser()` was called on user profile screen `dispose()` method if the no zendesk user was loggedIn .
This error did not affect functionality or UX in any way, it was just a minor annoyance while debugging.

## Requirements
_list any dependencies necessary to review the changes_
- _eg. make sure you reaplace your `.env` with the latest version_

## Type of change
- [ ] feature
- [ ] hotfix
- [x] fix
- [ ] refactor
- [ ] chore
- [ ] docs

## How to test?
Run the app as you usually do, go to User Profile tab and then go to any other tab, the uncaught exception should not appear on the console.

## How has this been tested?
Same as previous section

## Screenshots
N/A

## Checklist (for the reviewer)

- [ ] Does the code compile without errors or warnings?
- [ ] Does the PR is free of merge conflicts?
- [ ] Does the UI is responsive?
- [ ] Is the PR modular?

### CODE REVIEW

**Comment by SilvioArzeno** [comment]
@kiwis08 Please add Summary, How to Test el How this has been tested and maybe add a section for "Impacted Features" and put Zendesk under it. Nice PR Template btw. I will indeed steal it for future use

File: lib/src/features/user-profile/presentation/user_profile_screen.dart [hunk 1]
```diff
@@ -58,7 +58,9 @@ class _UserProfileScreenState extends ConsumerState<UserProfileScreen> {
   @override
   void dispose() {
     _refreshTimer.cancel();
-    ZendeskMessaging.logoutUser();
+    if (isLogin) {
+      ZendeskMessaging.logoutUser().onError((error, stacktrace) => debugPrint('Error logging out of Zendesk: ${error.toString()}'));
```
Comment (SilvioArzeno) [inline lib/src/features/user-profile/presentation/user_profile_screen.dart:62]: You can use safePrint here as well

**Review by SilvioArzeno — CHANGES_REQUESTED** [review]
Check out the comments I left on the review

File: lib/main.dart [hunk 2]
```diff
@@ -42,17 +42,19 @@ void main() async {
   if (Env.environment == Environment.stage.name) {
     HttpOverrides.global = new MyHttpOverrides();
   }
+
+  if (kReleaseMode) {
+    debugPrint = (String? message, {int? wrapWidth}) {};
+  }
+
```
[the remark this answers is not in the document]
Comment (SilvioArzeno) [reply lib/main.dart:49]: This safePrint is included in a AWS Package that I will add later on so nvm @kiwis08

File: lib/src/features/user-profile/presentation/user_profile_screen.dart (hunk 1 above)
Comment (SilvioArzeno) [reply lib/src/features/user-profile/presentation/user_profile_screen.dart:62]: nvm here as well

**Review by SilvioArzeno — APPROVED** [review]
LGTM

**Review by AlbertoMorillo — APPROVED** [review]

**Pull request merged**